cli/create.go

```diff
@@ -53,10 +53,10 @@ func (cc *CreateCommand) runCreate(args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create container: %v", err)
 	}
-
 	config.Image = ref.String()
-	if len(args) == 2 {
-		config.Cmd = strings.Fields(args[1])
+
+	if len(args) > 1 {
+		config.Cmd = args[1:]
```

Comment:
You need to test it,  it can work fine ?????
e.g.: 
pouch create image ls -l

Reply:
I remember that you told me before, while I forget this. Tomorrow I will have a try.

Reply:
Yes, I tried:
```
root@ubuntu:~/go/src/github.com/alibaba/pouch# pouch create --name foo registry.hub.docker.com/library/busybox:latest ls -a
container ID: 38304cda7803dcaca1c5a3fcaa8fb08e13f9392fd54906ff96407b8bf8bd3b1b, name: foo
```

The important thing I found is that we should add `flagSet.SetInterspersed(false)`. @skoo87